alexandria/core/tests/test_views.py

```diff
@@ -44,7 +46,7 @@ def test_anonymous_writing(db, document, client, settings, user, allow_anon, met
 
     resp = getattr(client, method)(url, data=data)
     assert (
-        resp.status_code == HTTP_201_CREATED or HTTP_200_OK
+        resp.status_code in {HTTP_201_CREATED, HTTP_200_OK}
```

Comment:
is there a particular rationale for the usage of `{` ... `}` or just to spice up things a little? ;)